dashboard/app/controllers/home_controller.rb

```diff
@@ -10,6 +10,7 @@ class HomeController < ApplicationController
 
   def set_locale
     set_locale_cookie(params[:locale]) if params[:locale]
+    set_partner_cookie(params['partner']) if params['partner']
```

Comment:
you can use the symbol :partner here (params is an indifferent hash so string/symbol means the same thing)